frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -1198,24 +1222,22 @@ export const funnelLogic = kea<funnelLogicType>({
             { visible }: { visible: boolean },
             breakpoint: BreakPointFunction
         ) => {
-            if (visible) {
+            if (visible && values.shouldReportCorrelationViewed) {
                 eventUsageLogic.actions.reportCorrelationViewed(values.filters, 0)
+                await breakpoint(10000)
+                eventUsageLogic.actions.reportCorrelationViewed(values.filters, 10)
             }
-
-            await breakpoint(10000)
-            eventUsageLogic.actions.reportCorrelationViewed(values.filters, 10)
         },
 
         [visibilitySensorLogic({ id: `${values.correlationPropKey}-properties` }).actionTypes.setVisible]: async (
             { visible }: { visible: boolean },
             breakpoint: BreakPointFunction
         ) => {
-            if (visible) {
+            if (visible && values.shouldReportPropertyCorrelationViewed) {
                 eventUsageLogic.actions.reportCorrelationViewed(values.filters, 0, true)
+                await breakpoint(10000)
+                eventUsageLogic.actions.reportCorrelationViewed(values.filters, 10, true)
             }
-
-            await breakpoint(10000)
```

Comment:
the problem you mentioned with "analyzed" wasn't that it was happening 10 seconds after scroll, but that after the "not visible" event was fired, we were still firing the delayed breakpoint 🤦